index.html

```diff
@@ -0,0 +1,1156 @@
+<!DOCTYPE html>
```

Comment:
This clone looks precisely the same as the original. It's awesome! :+1:

Reply:
You are absolutely right 👌 But please try to find some points to improve the code. 
Kindly check the code to look for errors and bugs and leaves nice comments containing change suggestions for each. Examples of issues the student may when performing the review:

Lack of [semantic HTML tags use](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Not applying [HTML & CSS best practices correclty](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Bugs missed the first time the project was built.